rules/utils/element.js

```diff
@@ -223,11 +224,11 @@ module.exports.hasProperties = function(node, properties, parentNode = null) {
       ];
     }
 
-    if (propertyChecks.validate && !propertyChecks.validate(propertyValue)) {
+    if (isFunction(propertyChecks.allowed) && !propertyChecks.allowed(propertyValue)) {
       return [
         ...results,
         {
-          message: `Property <${ propertyName }> of element of type <${ node.$type }> has invalid value`,
+          message: `Property <${ propertyName }> of element of type <${ node.$type }> must not have value of <${ propertyValue }>`,
```

Comment:
I'd rather like to avoid pasting property value to the output of the linter, as this may have undesirable outcome, e.g. 
```
Property <${ propertyName }> of element of type <${ node.$type }> must not have value of <aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa>
```
Preferably, we should control what gets to the output and not the creator of the diagram.

Reply:
I'd say we should output only what we control.

Reply:
I mean it's all good until somebody does `node.innerHTML = report.message`.